services/movie_session.py

```
@@ -0,0 +1,55 @@
+
+import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(movie_show_time: datetime.datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall,
+        movie=movie)
+    return movie_session
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet[MovieSession]:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        date_obj = datetime.datetime.strptime(session_date, "%Y-%m-%d").date()
+        queryset = queryset.filter(show_time__date=date_obj)
+    return queryset
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession | None:
```

Error: The return type annotation for get_movie_session_by_id is 'MovieSession | None', but MovieSession.objects.get will raise a DoesNotExist exception if no object is found, rather than returning None. Please update the return type annotation to 'MovieSession'.